tests/Unit/Elliptic/Systems/Elasticity/Test_Equations.cpp

```diff
@@ -29,15 +29,18 @@ namespace {
 
 template <size_t Dim>
 void primal_fluxes(
+    /* This wrapper function is needed to construct a
+     * constitutive relation for the random values of its parameters.
+     */
```

Comment:
Please use `//` for this comment (not `/* */`)

Reply:
Okay. I will make sure to read through the style guide again.